server/container_status.go

```diff
@@ -65,23 +65,17 @@ func (s *Server) ContainerStatus(ctx context.Context, req *types.ContainerStatus
 		cState = c.State()
 	}
 
-	created := c.CreatedAt().UnixNano()
+	resp.Status.CreatedAt = cState.Created.UnixNano()
+	resp.Status.StartedAt = cState.Started.UnixNano()
+	resp.Status.FinishedAt = cState.Finished.UnixNano()
```

Comment:
It looks like the startedAt and finishedAt timestamp might be `-6795364578871345152`(`time.Time{}.UnixNano()`)
https://github.com/cri-o/cri-o/blob/f47aeb6bf10cc62b4b5af2283fa507ddc5242191/vendor/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L925-L938
`time.Unix(0, -6795364578871345152).IsZero()` is false

Reply:
The current code should be correct

Reply:
Ah, yeah thank you for the wrap-up. This was a bit confusing.

Reply:
Let's keep the integration test.